### ISSUE TITLE
[Feature Request]: CodeMode: Add backup mode which uses only Letters, not digits

### ISSUE THREAD

**Issue by JanneVed** [Derpitron/Discord-OTP-Forcer#68]
### Description

When trying to use backup CodeMode it generates both number and letters instead of only letters.

### Minimal Reproduction

start the script in CodeMode: backup

### Error

Getting both nummbers and letters in the backup code

### Operating System Version

Windows 10 PRO 19044.2251

### Python Version

3.11

### Program Version 

7e17166037641096544368472b271e6e22784dc0

**Comment by LuXeZs** [comment]
Backup codes are made up of both letters and numbers.
![Discord_x8N7aAZCQY](https://github.com/Derpitron/Discord-OTP-Forcer/assets/49080760/b64e6ba5-fc95-4b26-ad0d-0883fd5c3b8b)

**Comment by JanneVed** [comment]
Hmm all that I remember from my codes was only letters. Have it been
changed or am I delusional?

Den mån 26 juni 2023 11:47LuXeZs ***@***.***> skrev:

**Comment by LuXeZs** [comment]
They might've changed but not 100% on that. If you want I can edit the code gen to only use letters if you think yours might be that.

**Comment by JanneVed** [comment]
Yeah I'm not sure now either. Would it be possible to add an extra for only
letters to be used, I have a memory of it only being letters for my codes.

Den mån 26 juni 2023 11:54LuXeZs ***@***.***> skrev:

**Comment by LuXeZs** [comment]
I've added a 3rd option to allow you to only use letters for the code, just use `backup_let` in the codemode section to use it.

**Comment by JanneVed** [comment]
Awesome thanks!

Den mån 26 juni 2023 15:40LuXeZs ***@***.***> skrev:
